### ISSUE TITLE
Installation wizard - Node setup test button - Substrate base-chain #200

### ISSUE THREAD

**Issue by zimaah** [SimplyVC/panic#207]
### User Story

As a node operator, I want to be able to verify that the Node exporter URL is correctly configured before my settings are finalized.

### Description

The scope of this task is **limited exclusively to the frontend aspect** of the test button associated with the nodes setup step - see #48.

### Requirements

Introduce a Test button for the Node exporter URL.

### Blocked by

Not dependent on another ticket to be tackled.

### Acceptance criteria

**Scenario**: Node operator is using the installation wizard to setup monitoring and alerting for a Substrate chain
**Given**: The node operator is in the Nodes setup step
**Then**: The node operator is presented with a Test button corresponding to the Node exporter URL per node he wants to set up.
